src/libcommon/utility/types.h

```diff
@@ -364,10 +366,31 @@ struct ItemType {
 
 enum class AppStateKey {
     // Adding a new key here requires to add it in insertDefaultAppState in parmsdbappstate.cpp
-    LastServerSelfRestart,
-    LastClientSelfRestart,
-
-    Unknown,  //!\ keep in last position (For tests) /!\\ Used for initialization, will throw error if used.
+    LastServerSelfRestartDate,
+    LastClientSelfRestartDate,
+    LastSuccessfulLogUploadDate,
+    LastLogUploadArchivePath,
+    /* Log upload status is a string with the first character being the status and the rest being the progress percentage
```

Comment:
This comment seems outdated.

Reply:
Yes, thanks you for pointing it out. I change it in commit: [1885599](https://github.com/Infomaniak/desktop-kDrive/pull/117/commits/18855991e78e0dd15df6c38db404f2a38f5c85eb)